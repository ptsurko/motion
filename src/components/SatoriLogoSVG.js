import React, { PureComponent } from 'react';

export default class SatoriLogoSVG extends PureComponent {
  render() {
    return (
      <svg className="phone-avatar" viewBox="0 0 36 31">
        <title>Satori Logo</title>
        <defs>
            <polygon id="path-1" points="0 30.3545182 126.785352 30.3545182 126.785352 0.0182127109 0 0.0182127109"></polygon>
        </defs>
        <g id="Symbols" stroke="none" strokeWidth="1" fillRule="evenodd">
            <g id="TopNavigation" transform="translate(-58.000000, -14.000000)">
                <g id="Satori-Logo" transform="translate(58.000000, 14.000000)">
                    <polygon id="Fill-1" points="10.0869175 0.253030205 15.7789195 10.0309794 32.1506761 10.0309794 26.458674 0.253030205"></polygon>
                    <polygon id="Fill-2" points="3.32435915 20.8113106 9.01636117 30.5892598 25.4134155 30.5892598 19.7214135 20.8113106"></polygon>
                    <polygon id="Fill-3" points="8.79282072 0.253030205 4.40889316 7.76906187 -7.66599598e-05 15.3286017 17.5854626 15.3286017"></polygon>
                    <polygon id="Fill-4" points="17.8896748 15.5127777 26.6823167 30.5883492 35.4749586 15.5127777"></polygon>
                </g>
            </g>
        </g>
      </svg>
    );
  }
}
