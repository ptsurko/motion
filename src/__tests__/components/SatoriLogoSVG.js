import React from 'react';
import renderer from 'react-test-renderer';
import SatoriLogoSVG from 'components/SatoriLogoSVG';

it('renders SatoriLogoSVG component correctly', () => {
  const tree = renderer.create(
    <SatoriLogoSVG />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
