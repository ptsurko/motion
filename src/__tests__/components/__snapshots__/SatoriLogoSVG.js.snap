// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SatoriLogoSVG component correctly 1`] = `
<svg
  className="phone-avatar"
  viewBox="0 0 36 31"
>
  <title>
    Satori Logo
  </title>
  <defs>
    <polygon
      id="path-1"
      points="0 30.3545182 126.785352 30.3545182 126.785352 0.0182127109 0 0.0182127109"
    />
  </defs>
  <g
    fillRule="evenodd"
    id="Symbols"
    stroke="none"
    strokeWidth="1"
  >
    <g
      id="TopNavigation"
      transform="translate(-58.000000, -14.000000)"
    >
      <g
        id="Satori-Logo"
        transform="translate(58.000000, 14.000000)"
      >
        <polygon
          id="Fill-1"
          points="10.0869175 0.253030205 15.7789195 10.0309794 32.1506761 10.0309794 26.458674 0.253030205"
        />
        <polygon
          id="Fill-2"
          points="3.32435915 20.8113106 9.01636117 30.5892598 25.4134155 30.5892598 19.7214135 20.8113106"
        />
        <polygon
          id="Fill-3"
          points="8.79282072 0.253030205 4.40889316 7.76906187 -7.66599598e-05 15.3286017 17.5854626 15.3286017"
        />
        <polygon
          id="Fill-4"
          points="17.8896748 15.5127777 26.6823167 30.5883492 35.4749586 15.5127777"
        />
      </g>
    </g>
  </g>
</svg>
`;
